meson.build

```diff
@@ -1,13 +1,13 @@
-project('spidermonkey-embedding-examples', 'cpp', version: 'esr90',
+project('spidermonkey-embedding-examples', 'cpp', version: 'esr91',
     meson_version: '>= 0.43.0',
-    default_options: ['cpp_std=c++17', 'warning_level=3'])
+    default_options: ['cpp_std=c++17', 'warning_level=1'])
 
 cxx = meson.get_compiler('cpp')
 
-args = ['-std=c++17', '-Wall']
+args = []
```

Comment:
We will still need `-std=c++17`, no?

Reply:
That is covered by the `cpp_std=c++17` above in the default_options (which is what meson was complaining about so I removed the line from args).

Reply:
Oh, right, I missed that.